actionpack/lib/action_controller/metal/params_wrapper.rb

```diff
@@ -231,16 +231,18 @@ def process_action(*args)
       if _wrapper_enabled?
         if request.parameters[_wrapper_key].present?
           wrapped_hash = _extract_parameters(request.parameters)
+          wrapped_request_hash = _extract_parameters(request.request_parameters)
         else
-          wrapped_hash = _wrap_parameters request.request_parameters
+          wrapped_hash = _wrap_parameters request.parameters
+          wrapped_request_hash = _wrap_parameters request.request_parameters
         end
 
         wrapped_keys = request.request_parameters.keys
         wrapped_filtered_hash = _wrap_parameters request.filtered_parameters.slice(*wrapped_keys)
```

Comment:
I think the `slice` stuff here is now unnecessary and incorrect - previously we only wrapped request parameters, so we had to filter out query and path parameters here. Now we're wrapping all parameters, so this can just be:

```ruby
wrapped_filtered_hash = _wrap_parameters request.filtered_parameters
```